.github/workflows/elvis_lint.yaml

```diff
@@ -0,0 +1,20 @@
+name: Elvis Linter
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          echo "$GITHUB_BASE_REF"
+          echo "$GITHUB_HEAD_REF"
+          set +e
```

Comment:
Please put the scripts to `elvis-check.sh` and run `./elvis-check.sh "$GITBHUG_BASE_REF" "$GITHUB_HEAD_REF"` here.